src/Registry.xml

```diff
@@ -2515,7 +2523,7 @@
 		<var name="verticalSalinity"			type="real"	dimensions="nBioLayers nCategories nCells Time"		name_in_code="verticalSalinity"		packages="pkgTracerZSalinity"/>
 	</var_struct>
 	<var_struct name="tracers_aggregate" time_levs="1">
-		<var name="iceAreaCell"			type="real"	dimensions="nCells Time"			name_in_code="iceAreaCell"/>
+		<var name="iceAreaCell"			type="real"	dimensions="nCells Time"	name_in_code="iceAreaCell"/>
```

Comment:
There's a few of these whitespace changes that made their way in to the Registry.xml. It feels niggling, but I don't think we should have these in the same commit even if they're needed.

Reply:
Corrected. Set of diff lines now much smaller.

I do the git push --force ... but it tells me 'Everything up-to-date'

Reply:
> Corrected. Set of diff lines now much smaller.
> 
> I do the git push --force ... but it tells me 'Everything up-to-date'

Hmmm that's odd. Could you point me to the directory?

Reply:
Nice, [eb5cf9f](https://github.com/EarthWorksOrg/mpas-seaice/pull/8/commits/eb5cf9f1fa424cb809bc985cb197353f0ef204df) has addressed this.